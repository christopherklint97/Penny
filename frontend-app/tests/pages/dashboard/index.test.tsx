import React from 'react'
import { render } from '../../testUtils'
import Dashboard from '../../../pages/dashboard/index'

describe('Dashboard page', () => {
  it('renders without crashing', function () {
    render(<Dashboard />)
  })

  it('matches snapshot', () => {
    const { asFragment } = render(<Dashboard />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
