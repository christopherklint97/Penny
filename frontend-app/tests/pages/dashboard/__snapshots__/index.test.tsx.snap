// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard page matches snapshot 1`] = `<DocumentFragment />`;
